javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -751,7 +751,7 @@ private predicate flowThroughCall(
 ) {
   exists(Function f, DataFlow::ValueNode ret |
     ret.asExpr() = f.getAReturnedExpr() and
-    calls(output, f) and // Do not consider partial calls
+    (calls(output, f) or callsBound(output, f, _)) and // Do not consider partial calls
```

Comment:
We have four occurrences of the pair `calls(output, f) or callsBound(output, f, _)` in this commit, are they worth moving into a predicate?